app/uk/gov/hmrc/eoricommoncomponent/frontend/views/layout.scala.html

```diff
@@ -25,14 +25,18 @@
 @import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
 @import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcLayout
 @import views.html.helper.CSPNonce
-
-@this(hmrcLayout: HmrcLayout, standardBetaBanner: StandardBetaBanner, trackingConsentSnippet: HmrcTrackingConsentSnippet, appConfig: AppConfig, reportAProblemLink: HmrcReportTechnicalIssue, hmrcTimeoutDialog: HmrcTimeoutDialog)
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukBackLink
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.backlink.BackLink
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers
+@this(hmrcLayout: HmrcLayout, standardBetaBanner: StandardBetaBanner, govukBackLink: GovukBackLink, trackingConsentSnippet: HmrcTrackingConsentSnippet, appConfig: AppConfig, reportAProblemLink: HmrcReportTechnicalIssue, hmrcTimeoutDialog: HmrcTimeoutDialog)
 
 @(
     title: String,
     countriesInCountryPicker: CountriesInCountryPicker = NoCountriesInCountryPicker,
     form: Option[Form[_]] = None,
-    suppressTelephoneNumberDetection: Boolean = true
+    suppressTelephoneNumberDetection: Boolean = true,
+        backLink: Option[String] = None
```

Comment:
I think backLink shows a a 'back' link that over rides the default back button.
It's used to prevent users going 'back' to a page with logic that isn't stand alone i.e. what is displayed relies on info that is retrieved or created by previous pages.
The backLink can be set to return the user to a page that makes functional and logical sense in the service